projects/ad4630_fmc/zed/README.md

```diff
@@ -37,81 +37,96 @@ The overwritable parameters from the environment are:
 - DDR_EN: in echo and master clock mode, the SDI lines can have Single or Double data rates
   - 0 - MISO runs on SDR (default)
   - 1 - MISO runs on DDR
+- NO_REORDER: removes the spi_axis_reorder from system for CAPTURE_ZONE = 1 and
+  NUM_OF_SDI = 1 (ad4030) or NUM_OF_SDI = 2 (ad4630) and directly connects the SPI
+  Engine to DMA
+  - 0 - spi_axis_reorder present (default)
+  - 1 - spi_axis_reorder removed
 
 ### Example configurations
 
-#### Clock mode SPI, MOSI lines 4, Capture zone 2, SDR (default)
+#### Clock mode SPI, MISO lines 4, Capture zone 2, SDR (default)
 
 This specific command is equivalent to running `make` only:
 
 ```
-make CLK_MODE=0 NUM_OF_SDI=4 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=0 NUM_OF_SDI=4
 ```
 
 Corresponding device trees:
 - [zynq-zed-adv7511-ad4630-24.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-ad4630-24.dts)
-- [zynq-zed-adv7511-adaq4216.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-adaq4216.dts)
-- [zynq-zed-adv7511-adaq4220.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-adaq4220.dts)
-- [zynq-zed-adv7511-adaq4224-24.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-adaq4224-24.dts)
+- [zynq-zed-adv7511-ad4630-16.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-ad4630-16.dts)
 
-#### Clock mode SPI, MOSI lines 2, Capture zone 2, SDR
+#### Clock mode SPI, MISO lines 2, Capture zone 2, SDR
 
 ```
-make CLK_MODE=0 NUM_OF_SDI=2 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=0 NUM_OF_SDI=2
```

Comment:
Same here and for the other instances as well

Reply:
Yes, I agree, but it is clearer to the user. What if the default configuration in system_project.tcl changes? And running this command would in fact build with another "capture zone" or with "DDR enabled"?

Reply:
Fixed as you wish!